contributing_to_docs/doc_guidelines.adoc

```diff
@@ -63,6 +63,25 @@ If using section anchors, they must be all lowercase letters, with no line space
 ----
 
 == Links, hyperlinks, and cross references
+Whenever possible use inline embedded links to other topics or other references.
+
+Example of recommended usage where possible:
+
+----
+Rollbacks revert part of an application back to a previous deployment. Rollbacks can be performed using the REST API or
+the link:cli.html#deployment-rollbacks[OpenShift CLI].
+----
+
+This will render as so:
+
+Rollbacks revert part of an application back to a previous deployment. Rollbacks can be performed using the REST API or the link:cli.html#deployment-rollbacks[OpenShift CLI].
+
+As compared to this, which is not recommended:
+
+----
+Rollbacks revert an application back to a previous deployment and can be performed using the REST API or
+the OpenShift CLI. See the link:cli.html#deployment-rollbacks[CLI documentation] for more details.
```

Comment:
I'm confused - your "don't do it this way" example is functionally identical to the "do it this way" example.

I think the "don't do it this way" example that you want is:

    See the http://docs.openshift.org/latest/using_openshift/cli.html#deployment-rollbacks[Openshift CLI]